### DISCUSSION TITLE
Entity picture for tab header

### DISCUSSION THREAD

**Discussion by palves** [kinghat/tabbed-card#89] [Feature Requests]
Currently you can have either a label or an icon as tab attribute.  IWBN if we could also choose to use an entity picture or a custom picture.  

My use case is that I have one map card per tab, with each map tracking a person's location in the last 24 hours. I would like to show each person's entity picture in the tab header instead of a label or an icon.  Failing that, letting one point at a custom picture would also work.

**Comment by kinghat** [comment]
converted to a feature request. if implemented, it would take the place of an icon. you wouldnt be able to have both. no timeline on implementation.
